website/docs/r/incident_workflow.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_incident_workflow"
+sidebar_current: "docs-pagerduty-resource-incident-workflow"
+description: |-
+  Creates and manages an incident workflow in PagerDuty.
+---
+
+# pagerduty\_incident\_workflow
+
+An Incident Workflow is a series of steps which can be executed on an incident.
```

Comment:
For being consistent with the rest of the resources' documentation, please use the **Incident Workflow** resource name for referencing the documentation of the product. e.g.: `[Incident Workflow](https://support.pagerduty.com/docs/<product-docs-page-here>`

Reply:
Added